invenio_files_rest/models.py

```diff
@@ -964,10 +966,20 @@ def validate_key(self, key, key_):
         """Validate key."""
         return validate_key(key_)
 
-    def __repr__(self):
-        """Return representation of location."""
-        return '{0}:{2}:{1}'.format(
-            self.bucket_id, self.key, self.version_id)
+    def __unicode__(self):
```

Comment:
Instead you could use the `@python_2_unicode_compatible` decorator and just create a `__str__`  function returning unicode and the `__repr__` returning the `__str__` to avoid manually controlling the python version.

Reply:
I wanted to use this decorator, but @slint suggested not to, since it is a class decorator, not a function one and could affect some other class behaviour.